gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Dataset.java

```diff
@@ -212,69 +211,23 @@ public boolean delete() {
    * @throws BigQueryException upon failure
    */
   public Table get(String table, BigQuery.TableOption... options) {
-    BaseTableInfo tableInfo =
+    TableInfo tableInfo =
         bigquery.getTable(TableId.of(info.datasetId().dataset(), table), options);
     return tableInfo != null ? new Table(bigquery, tableInfo) : null;
   }
 
   /**
-   * Creates a new simple table in this dataset.
+   * Creates a new table in this dataset.
    *
    * @param table the table's user-defined id
-   * @param schema the table's schema
+   * @param definition the table's definition
    * @param options options for table creation
    * @return a {@code Table} object for the created table
    * @throws BigQueryException upon failure
    */
-  public Table create(String table, Schema schema, BigQuery.TableOption... options) {
-    BaseTableInfo tableInfo = TableInfo.of(TableId.of(info.datasetId().dataset(), table), schema);
-    return new Table(bigquery, bigquery.create(tableInfo, options));
-  }
-
-  /**
-   * Creates a new view table in this dataset.
-   *
-   * @param table the table's user-defined id
-   * @param query the query used to generate the table
-   * @param functions user-defined functions that can be used by the query
-   * @param options options for table creation
-   * @return a {@code Table} object for the created table
-   * @throws BigQueryException upon failure
-   */
-  public Table create(String table, String query, List<UserDefinedFunction> functions,
-      BigQuery.TableOption... options) {
-    BaseTableInfo tableInfo =
-        ViewInfo.of(TableId.of(info.datasetId().dataset(), table), query, functions);
-    return new Table(bigquery, bigquery.create(tableInfo, options));
-  }
-
-  /**
-   * Creates a new view table in this dataset.
-   *
-   * @param table the table's user-defined id
-   * @param query the query used to generate the table
-   * @param options options for table creation
-   * @return a {@code Table} object for the created table
-   * @throws BigQueryException upon failure
-   */
-  public Table create(String table, String query, BigQuery.TableOption... options) {
-    BaseTableInfo tableInfo =  ViewInfo.of(TableId.of(info.datasetId().dataset(), table), query);
-    return new Table(bigquery, bigquery.create(tableInfo, options));
-  }
-
-  /**
-   * Creates a new external table in this dataset.
-   *
-   * @param table the table's user-defined id
-   * @param configuration data format, location and other properties of an external table
-   * @param options options for table creation
-   * @return a {@code Table} object for the created table
-   * @throws BigQueryException upon failure
-   */
-  public Table create(String table, ExternalDataConfiguration configuration,
+  public Table create(String table, BaseTableDefinition definition,
```

Comment:
Fine as is (much better now), so more an FYI but per my earlier rename suggestion my preference
would have been to go with BaseTableDefinition -> TableDefinition and what is now TableDefinition -> StandardTableDefinition as I feel it is slightly more consistent with JobConfiguration.